src/components/ShortcutTile.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { useClient, useFetchShortcut } from 'cozy-client'
+import get from 'lodash/get'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import useBreakpoints from 'cozy-ui/transpiled/react/hooks/useBreakpoints'
+
+const ShortcutTile = ({ file }) => {
+  const client = useClient()
+  const { shortcutInfos } = useFetchShortcut(client, file._id)
+  const url = get(shortcutInfos, 'data.attributes.url', '#')
+  const shortcurtIcon = get(shortcutInfos, 'data.attributes.metadata.icon')
+  const label = file.name.substr(0, file.name.lastIndexOf('.'))
```

Comment:
Do we want to get the name before the .url? If yes why not using https://github.com/cozy/cozy-libs/blob/master/packages/cozy-doctypes/src/File.js#L82 for instance ?

Reply:
I had a look at that model… and somehow missed that? Will use 👍